addon/addon/models/context-aware-bounds.ts

```diff
@@ -58,9 +58,7 @@ export default class ContextAwareBounds {
     let { element, parent } = this;
 
     if (!parent) {
-      // TODO: in the case of an interrupted orphan, there's no defined parent (in the sprite tree).
-      //  The below is a hotfix to send back the correct bounds for the situation. We need a better solution.
-      return this.relativeToContext;
+      throw new Error('Could not access parent DOMRect in relativeToParent');
```

Comment:
Throwing this error to make it easier to find cases where parent is undefined, removing comment since I was mistaken about the cause in this case - it was because of assigning a `finalBounds` on a removed sprite.

Reply:
Yeah seems good to have this.